### DISCUSSION TITLE
[Question] How to remove blur behind waybar?

### DISCUSSION THREAD

**Discussion by WorldTeacher** [prasanthrangan/hyprdots#1041] [Q&A]
I recently found this repo and am planning to use it as my baseline for hyprland. I have already created a waybar setup I want to use, but don't like the blur added below the waybar.
I replaced the waybar folder with mine and restarted waybar, but noticed the blur beneath the bar. 
 Is there a way to remove it?

**Comment by WorldTeacher** [comment]
![IMG_20240309_082721.jpg](https://github.com/prasanthrangan/hyprdots/assets/41587052/acf63386-d2d0-461a-8cfd-f903d81621b2)

Image as example

**Comment by kRHYME7** [comment]
@WorldTeacher 

https://wiki.hyprland.org/Configuring/Window-Rules/#rules

```
# layerrule = unset, waybar
```



Note that the configuration to add layerrules are written on each theme on this directory  ``` ~/.config/hypr/themes  ```

Hyprland configs are dynamic, therefore you can override it on the ./userprefs.conf 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/9d64b5f8-2b40-490a-8cf2-679da9968afb)
 
Example

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/8f9dbbd0-9073-41d6-9045-34a1f10763fe)

**Reply by WorldTeacher** [reply]
Thanks, I tried setting it in userprefs, but it seems thst does not apply the setting on my system, will try after a clean install on my main drive. Otherwise I'll just remove the blur from each theme
